test_apps/smokeTests/testResults/cf_baseline.txt

```diff
@@ -60,4 +61,4 @@ Time Coordinates:
 
 Time coordinate variable name: time
 Number of time steps: 1
-Elapsed time: 0.318985
+Elapsed time: 0.159113
```

Comment:
@sgpearse can we remove the output of elapsed time when creating these reference files so don't keep running into this?

Reply:
This wasn't the issue - diff found that there was a new std::endl in one of the printouts, as well as another line reporting the number of threads used.  The elapsed time is not compared in the diff, noted in [smokeTests.py:173](https://github.com/NCAR/VAPOR/blob/b457c5c5f59a52f28e8d5f7d3fcb8820c6d6915c/test_apps/smokeTests/smokeTests.py#L173)